app/mailers/dev_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class DevMailer < SiteMailer
+
+  def pp(object:, to:, subject:)
```

Comment:
It originally called pretty print.  Wanted a way to pretty format an object for use by developers.  `pp` turned out to not format how I wanted, so I switched to `awesome_print`.  

Will choose a better name.